.sdkmanrc

```diff
@@ -0,0 +1,4 @@
+# Enable auto-env through the sdkman_auto_env config
```

Comment:
Why is this needed?

Reply:
This is to force sdkman to switch to Java 11 to run this project. Until we make this project compatible with Java 17 (or higher), this is needed. It's easy to forget that this project runs on Java 11. May be this should be includes in the README too.